polytracker/include/polytracker/passes/DataFlowSanitizer.h

```diff
@@ -0,0 +1,31 @@
+//===- DataFlowSanitizer.h - dynamic data flow analysis -------------------===//
```

Comment:
This is a modified copy of the llvm-file. Is this the appropriate way of adding copyright notices etc. Should we note that this is a modified version? I'm assuming we should be keeping this copyright header, but maybe add some additional comments.

Reply:
I don't really have an opinion on this. Maybe @ESultanik could chime in?

Reply:
We could consider adding the git commit hash of the version from which we forked, but I think this is otherwise fine. We might want to update our base README with copyright notices from forked/vendored LLVM code if we haven't already done so.